script.js

```diff
@@ -71,6 +72,18 @@ function resetMenu() {
   }
 }
 
+function menuLinkIsClicked() {
+  menuLinks.forEach((element) => {
+    element.addEventListener("click", () => {
+      menuClicked = false;
+      deactivateMenuIcon();
+      closeMenu();
+    });
+  });
```

Comment:
Since you are attaching event listeners here to each menu item you do not need to extract this into a function and then call it immediately. This is a website wide event attachment and you are not reusing this logic elsewhere, therefore, it is ok to NOT wrap it with a function.